packages/components/tooltip/src/use-tooltip.ts

```diff
@@ -123,6 +125,7 @@ export function useTooltip(originalProps: UseTooltipProps) {
     onClose,
     motionProps,
     classNames,
+    shouldBlockScroll = false,
```

Comment:
the default value is `true` according to the docs.

Reply:
Changes Done